### PR TITLE
Add el8toel9 check support for systems with SAP Hana 

### PR DESCRIPTION
**Pull request by vinzenz**
This PR moves checksaphana to common and adjusts it for it to comply with the requirements for el7toel8 and el8toel9 upgrades.

### CODE REVIEW

File: repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py [hunk 1]
```diff
@@ -8,7 +8,8 @@
 
 SAP_HANA_MINIMAL_MAJOR_VERSION = 2
 SAP_HANA_RHEL8_REQUIRED_PATCH_LEVELS = ((5, 54, 0),)
-SAP_HANA_MINIMAL_VERSION_STRING = 'HANA 2.0 SPS05 rev 54 or later'
+SAP_HANA_RHEL9_REQUIRED_PATCH_LEVELS = ((5, 59, 4), (6, 63, 0))
+SAP_HANA_MINIMAL_VERSION_STRING = 'HANA 2.0 SPS05 rev 59.04 or later, or SPS06 rev 63 or later'
```
Comment (alexxa) [inline repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py:12]: This is a 8to9 minimal_version_string. For 7to8 it should stay original: 'HANA 2.0 SPS05 rev 54 or later'

File: repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py [hunk 2]
```diff
@@ -170,6 +175,15 @@ def version2_check(info):
 def platform_check():
     """ Creates an inhibitor report in case the system is not running on x86_64 """
     if not architecture.matches_architecture(architecture.ARCH_X86_64):
+        if version.get_target_major_version() == '8':
+            elink = reporting.ExternalLink(
+                url='https://access.redhat.com/solutions/5533441',
+                title='How do I upgrade from Red Hat Enterprise Linux 7 to Red Hat Enterprise Linux 8 with SAP HANA')
+        else:
+            elink = reporting.ExternalLink(
+                url='https://access.redhat.com/solutions/5533441',
```
Comment (alexxa) [inline repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py:184]: for 8to 9 it should be: url='https://access.redhat.com/solutions/6980855'

**Review by alexxa — CHANGES_REQUESTED** [review]

File: repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py [hunk 3]
```diff
@@ -8,7 +8,8 @@
 
 SAP_HANA_MINIMAL_MAJOR_VERSION = 2
 SAP_HANA_RHEL8_REQUIRED_PATCH_LEVELS = ((5, 54, 0),)
```
Comment (alexxa) [inline repos/system_upgrade/common/actors/checksaphana/libraries/checksaphana.py:10]: (5, 54, 0) is accurate if the upgrade is from 7.9 to 8.2
if it's from 7.9 to 8.6, then HANA must be withing a version of  [59.02; 59.05]

**Review by alexxa — APPROVED** [review]

**Comment by alexxa** [comment]
LGTM, thanks a lot @vinzenz

**Review by pirat89 — APPROVED** [review]
lgtm. keeping the testing on sap team (@alexxa )

**Comment by alexxa** [comment]
Inhibitors confirmed for 7.9 to 8.2/86 and 8.6 to 9.0 on pr978.17.g7cf7d8c

**Pull request merged**